test/Distributed/Runtime/distributed_actor_custom_executor_availability.swift

```diff
@@ -38,7 +38,7 @@ import Distributed
       }
 
       tests.test("5.7 actor, 5.9 executor property => no custom executor") {
-        expectCrashLater(withMessage: "Incorrect actor executor assumption; Expected MainActor executor")
+        expectCrashLater()
```

Comment:
The new semantics generate the crash in the `swift_task_isCurrentExecutorImpl` function, which occurs before the message is emitted.